CMakeLists.txt

```diff
@@ -343,11 +343,14 @@ if(MIGRAPHX_USE_ROCBLAS)
     list(APPEND PACKAGE_DEPENDS rocblas)
 endif()
 
+rocm_package_add_deb_dependencies(DEPENDS "hip-dev")
```

Comment:
Do we need do add this as shared dependency "rocm_package_add_dependencies(**SHARED_DEPENDS** "